src/phoenix/trace/utils.py

```diff
@@ -18,6 +18,16 @@ def is_jsonl_file(file_path: str) -> bool:
     return False
 
 
+def is_csv_file(file_path: str) -> bool:
+    """
+    Check if the given file is a csv file.
+    """
+    file_extension = os.path.splitext(file_path)[-1]
+    if file_extension == ".csv":
+        return True
+    return False
```

Comment:
this utility feels out of place, though it looks like is_jsonl was added above. Feels largely WET in a way that's not quite composable in a nice way (e.g. it seems like you just need a function that parses a file extension...)

Reply:
Implemented a function `parse_file_extension` instead